datadog-observability/README.md

```diff
@@ -0,0 +1,19 @@
+# Send Vector logs and metrics to Datadog
+
+In this demo, Vector is running a simple topology in which the `generator` source emits random JSON
+logs twice per second and sends those logs off to stdout via the `console` sink. But even this
+simple pipeline is enough to generate internal logs and metrics for Vector (via the eponymous
+sources).
+
+To run this scenario, make sure you have the `DD_API_KEY` environment variable set to your Datadog
+API key and then run:
+
+```bash
+docker compose up --detach --remove-orphans
```

Comment:
we could remove the `--remove-orphans` to kill this simple (and avoid removing any non-related container)